### PR TITLE
PROJ-177 Upgrade fontawesome to 5.15.0

### PR DESCRIPTION
**Pull request by LucWollants**
### Changed
- Upgraded to fontawesome 5.15.0
---
Ticket: https://jira.uitdatabank.be/browse/PROJ-177

### CODE REVIEW

**Review by simon-debruijn — APPROVED** [review]
Was a struggle to get right, but looks good 😉

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@
 config.json
 /app/scripts/config.js
 .DS_Store
+/app/fonts/fa-*
```
Comment (bertramakers) [inline .gitignore:10]: _This one weird trick will make 2000 lines of code disappear from your git repository!_

**Review by bertramakers — APPROVED** [review]
I don't have this running locally but the code changes themselves make sense :) Nice work!

File: .gitignore (hunk 1 above)
Comment (LucWollants) [reply .gitignore:10]: I think the removal of 2000 lines of code is not related to this change, it's related to running the `npm install` again with `node v14` and committing that updated lock file.

The excluded fonts are the new `fontawesone` files which were never added before.

Comment (bertramakers) [reply .gitignore:10]: Ah okay! It was just a joke though, about those ads about "this one weird trick will ..." 😉

Comment (LucWollants) [reply .gitignore:10]: Ok, that is "new" for me!

**Pull request merged**